submissions/zalevskyi/friends-app/app.js

```diff
@@ -0,0 +1,187 @@
+const USERS_DATA = []
+const USERS_QUANTITY = 30
```

Comment:
Why it is global also?

Reply:
`const USERS_QUNTITY` was global to understand easily this parameter of implementation in future.
It was easy to find answer to possible question "Why there is only 30 users shown?"
Now I have moved it into `function connectToServer()`. Or I can move it even lower to `function fetchUserData()`. And in this way it becomes more like hard coded magic number